SPECS/swig/swig.spec

```diff
@@ -52,6 +51,10 @@ make %{?_smp_mflags} check
 %{_libdir}/ccache
 
 %changelog
+* Mon Mar 15 2021 Henry Li <lihl@microsoft.com> - 4.0.2-1
```

Comment:
Since you're already here, could you check if the license information for this package is correct and add "License verified" to the changelog once that's done?

What to check:

- The contents of the `LICENSE` file say it's GPLv3 and not a different license, or a combination of multiple licenses, or something else entirely. If the info doesn't match, the `License` tag needs to be updated accordingly.
- There are no more files related to licensing in the source. If there are, we need to add them to the `%license` macro and update the `License` tag as well.

Reply:
Fixed

Reply:
Looks good, thank you! But please include the phrase "License verified" or "Verified license" in the changelog - we're trying to follow this convention to ease checking what we've already verified and what not.

Reply:
I think i already did. Put it in the first line after "Upgrade to version 4.0.2"

Reply:
Ah, completely missed that, sorry and thank you!:)